### DISCUSSION TITLE
Phantom Wallet Sometimes Not Recognized

### DISCUSSION THREAD

**Discussion by jackbl1** [Web3Auth/Web3Auth#827] [Help]
I've integrated web3Auth into a Solana app that I'm building and I noticed that occasionally my Phantom wallet isn't recognized as an external wallet option to connect. I'm having trouble consistently reproducing the issue, but just wondering if anyone else has experienced this, or if the team is aware of the issue?

The UX would appear like this, still recognizing the Torus wallet, just without the Phantom wallet option. It usually will resolve when I refresh the page.

<img width="380" alt="Screen Shot 2022-11-05 at 2 57 31 PM" src="https://user-images.githubusercontent.com/23587613/200136484-1a5f643a-0fc5-4b4c-8855-99848942dcdb.png">

**Comment by shahbaz17** [comment]
Hey @jackbl1

It could be because of the slow internet connection. Can you try to reproduce it using a slow internet connection?
